components/style/gecko/wrapper.rs

```diff
@@ -96,10 +92,36 @@ fn from_opaque_style_data(d: *mut OpaqueStyleData) -> *mut NonOpaqueStyleData {
 pub struct GeckoNode<'ln>(pub &'ln RawGeckoNode);
 
 impl<'ln> GeckoNode<'ln> {
+    fn from_content(content: &'ln nsIContent) -> Self {
+        use std::mem;
+        GeckoNode(unsafe { mem::transmute(content) })
+    }
+
+    fn node_info(&self) -> *mut structs::NodeInfo {
```

Comment:
Yep, at first I thought we might want to mutate it and didn't want to discuss aliasing issues, but it didn't happen.